fpdf/fpdf.py

```diff
@@ -505,6 +509,20 @@ def __init__(
         # page number -> array of 8 × n numbers:
         self.text_quad_points = defaultdict(list)
 
+    def write_html(self, text, *args, **kwargs):
+        """Parse HTML and convert it to PDF"""
+        kwargs2 = vars(self)
+        # Method arguments must override class & instance attributes:
+        kwargs2.update(kwargs)
+        h2p = self.HTML2FPDF_CLASS(self, *args, **kwargs2)
+        warnings.warn(
```

Comment:
Could you please raise this warning only if `self.HTML2FPDF_CLASS != HTML2FPDF`?

Reply:
fixed at f0ef001116c48ac221fd2af92c8aaf1897a35011